dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -503,9 +499,7 @@ private WorkflowVersion setupWorkflowVersionsHelper(Workflow workflow, Triple<St
             } else {
                 // Legacy version refresh of Dockstore.yml workflow, so use existing path for version (instead of default path)
                 if (!existingDefaults.containsKey(versionName.get())) {
-                    String msg = "Cannot refresh version " + versionName.get() + ". Only existing legacy versions can be refreshed.";
-                    LOG.error(msg);
```

Comment:
Ditto, in general think that while user error doesn't rise to the level of warn or error. 
It can be useful while debugging